addons/account_fleet/views/fleet_vehicle_log_services_views.xml

```diff
@@ -13,10 +13,17 @@
                         type="object"
                         class="oe_stat_button"
                         icon="fa-pencil-square-o"
-                        invisible="not account_move_line_id.move_id">
+                        invisible="not account_move_line_id.move_id or not account_move_line_id">
                         <div class="o_field_widget o_stat_info">
-                            <span class="o_stat_text text-success" invisible="account_move_state != 'posted'" title="Service's Bill"> Service's Bill</span>
-                            <span class="o_stat_text text-warning" invisible="account_move_state != 'draft'" title="Service's Bill"> Service's Bill</span>
+                            <span class="o_stat_text text-success"
+                                invisible="account_move_state != 'posted'"
+                                title="Service's Bill"> Service's Bill</span>
+                            <span class="o_stat_text text-warning"
+                                invisible="account_move_state != 'draft'"
+                                title="Service's Bill"> Service's Bill</span>
+                            <span class="o_stat_text"
+                                invisible="account_move_state in ['draft', 'posted']"
```

Comment:
there are 3 states: 'draft', 'posted', 'cancel'
shouldn't the service log disappear if the move is not posted?
I guess you obviously want to display it anyway, but then shouldn't 'draft' and 'cancel' be `text-warning`? if not, `account_move_state != 'cancel'` will be more uniform?

Reply:
I made it like this mostly for flexibility; if somehow a new state is added, be it in standard or in a custo, we would still have the text displayed rather than an empty space :) I guess using warning for draft and cancel makes sense, I'll change that.